mlflow/java/src/main/java/org/mlflow/sagemaker/LeapFrameSchema.java

```diff
@@ -0,0 +1,92 @@
+package org.mlflow.sagemaker;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import org.mlflow.utils.SerializationUtils;
+
+/** Representation of the dataframe schema that an {@link MLeapPredictor} expects inputs to have */
+@JsonIgnoreProperties(ignoreUnknown = true)
+class LeapFrameSchema {
+  private static final String LEAP_FRAME_KEY_ROWS = "rows";
+  private static final String LEAP_FRAME_KEY_SCHEMA = "schema";
+
+  @JsonIgnoreProperties(ignoreUnknown = true)
+  static class SchemaField {
+    @JsonProperty("name")
+    private String name;
+  }
+
+  @JsonProperty("fields")
+  private List<SchemaField> fields;
+
+  private String text;
+  private List<String> orderedFieldNames;
+
+  /**
+   * Loads a leap frame schema from a JSON-formatted file
+   *
+   * @param filePath The path to the JSON-formatted schema file
+   */
+  protected static LeapFrameSchema fromPath(String filePath) throws IOException {
```

Comment:
Should be package-private static, i.e., just `static`.

Reply:
Done!